### PR TITLE
Fix: Removing ESLint errors

### PR DESCRIPTION
**Pull request by Furaha-Justine**
**What does this PR do?**

Removing ESLint errors 

**Description of Task to be completed?**

Adding configurations to handle ESLint errors.

**What are the relevant pivotal tracker/Trello stories?**

**Screenshots (if appropriate)**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -42,6 +42,12 @@
           "ClassDeclaration": true
         }
       }
-    ]
+    ],
+    "prefer-destructuring":"off",
+    "no-console":"off",
```
Comment (munezerobagira) [inline .eslintrc.json]:
```suggestion
```
This rule ensure that no console log are shipped into production. Mostly console is used for debbugging purpose, if want to log some information you can use loggers like winston
https://github.com/winstonjs/winston

File: .eslintrc.json [hunk 2]
```diff
@@ -42,6 +42,12 @@
...
+    "no-unused-expressions": "off",
+    "@typescript-eslint/no-explicit-any": "off",
```
Comment (munezerobagira) [inline .eslintrc.json]: This rule ensures that typescript is used correctly, so its not ideal to remove it
```suggestion
```

File: .eslintrc.json [hunk 3]
```diff
@@ -42,6 +42,12 @@
...
+    "strict":"off",
```
Comment (munezerobagira) [inline .eslintrc.json]:
```suggestion
```
this is very important too

**Review by munezerobagira — CHANGES_REQUESTED** [review]

**Review by munezerobagira — APPROVED** [review]

File: .eslintrc.json [hunk 4]
```diff
@@ -42,6 +42,10 @@
           "ClassDeclaration": true
         }
       }
-    ]
+    ],
+    "prefer-destructuring": "off",
+    "no-unused-expressions": "off",
+    "@typescript-eslint/no-unused-vars": "off",
+    "@typescript-eslint/no-namespace": "off"
```
Comment (mpanoathos) [inline .eslintrc.json:49]: Nice

**Review by mpanoathos — APPROVED** [review]

**Review by AngeloChristian1 — APPROVED** [review]

**Review by EricNiyo73 — APPROVED** [review]

**Review by Bananayosostene — APPROVED** [review]

**Pull request merged**